internal/pkg/crawl/crawl.go

```diff
@@ -147,8 +150,8 @@ func (c *Crawl) Start() (err error) {
 	// to exit Zeno, like CTRL+C
 	go c.setupCloseHandler()
 
-	// Initialize the frontier
-	frontierLoggingChan := make(chan *frontier.FrontierLogMessage, 10)
+	// Initialize the frontier components
```

Comment:
if the package was renamed queue, maybe the `frontier` naming isn't relevant anymore?